doc/specs/application-auth-token.md

```diff
@@ -20,26 +35,79 @@ A semver string specifying the spec version under which this AAT needs to be int
 
 > type: `string`
 
-The application will sign a hash of the `message` property within this token with the specified `appPubKey` and
-corresponding private key.
+The application will sign a hash of the `message` property within this token with
```

Comment:
I find the `message` reference confusing: rephrasing this to say the application will sign a hash of all the other fields of the AAT (or even naming all the fields that are part of the signed bytes slice) would be easier to read.

There is a single reference to `message` in a diagram on this page, and that is SHA256 hash of the encoded AAT.